Assets/Scripts/Models/Tile.cs

```diff
@@ -112,7 +112,7 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovmentModifier { get; set; }
+    public float MovementModifier { get; set; }
```

Comment:
A public get and private set should work to be able to read from Lua. I don't think that we need to set it from lua?